pycbc/events/stat.py

```diff
@@ -44,8 +44,8 @@ def get_newsnr(trigs):
     numpy.ndarray
         Array of newsnr values
     """
-    dof = 2. * trigs['chisq_dof'] - 2.
-    newsnr = events.newsnr(trigs['snr'], trigs['chisq'] / dof)
+    dof = 2. * trigs['chisq_dof'][:] - 2.
```

Comment:
so here, trigs can be either a dictionary of ndarrays but also a h5py *group* which then yields *datasets* which then have to be indexed on [:] to give numpy arrays.  hence will ask for one tweaking of the documentation ..

Reply:
i.e. in the description of 'trigs' above add that it can also be an h5py group with datasets analogous to the required numpy arrays